### PR TITLE
Some block constants

### PR DESCRIPTION
**Pull request by IotaBread**

### CODE REVIEW

File: mappings/net/minecraft/block/AbstractFireBlock.mapping [hunk 1]
```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/class_4770 net/minecraft/block/AbstractFireBlock
 	FIELD field_22088 damage F
 	FIELD field_22498 BASE_SHAPE Lnet/minecraft/class_265;
+	FIELD field_31008 ENTITY_ON_FIRE_FOR I
```
Comment (liach) [inline mappings/net/minecraft/block/AbstractFireBlock.mapping]: Maybe `SET_ON_FIRE_TICKS`?

File: mappings/net/minecraft/block/TargetBlock.mapping [hunk 2]
```diff
@@ -1,5 +1,7 @@
 CLASS net/minecraft/class_4850 net/minecraft/block/TargetBlock
 	FIELD field_22428 POWER Lnet/minecraft/class_2758;
+	FIELD field_31263 PROYECTILE_POWER_DELAY I
```
Comment (liach) [inline mappings/net/minecraft/block/TargetBlock.mapping]:
```suggestion
	FIELD field_31263 PROJECTILE_POWER_DELAY I
```

Comment (IotaBread) [reply mappings/net/minecraft/block/TargetBlock.mapping]: I always write it with "y" as in Spanish lol

File: mappings/net/minecraft/block/AbstractFireBlock.mapping (hunk 1 above)
Comment (IotaBread) [reply mappings/net/minecraft/block/AbstractFireBlock.mapping]: The constant is in seconds, so I named it `SET_ON_FIRE_SECONDS`

File: mappings/net/minecraft/block/TargetBlock.mapping [hunk 3]
```diff
@@ -1,5 +1,7 @@
 CLASS net/minecraft/class_4850 net/minecraft/block/TargetBlock
 	FIELD field_22428 POWER Lnet/minecraft/class_2758;
+	FIELD field_31263 PROJECTILE_POWER_DELAY I
+	FIELD field_31264 OTHER_POWER_DELAY I
```
Comment (IotaBread) [inline mappings/net/minecraft/block/TargetBlock.mapping]: What should I name this? I don't think `OTHER` (as in other entities) fits

Comment (liach) [inline mappings/net/minecraft/block/TargetBlock.mapping]:
```suggestion
	FIELD field_31263 PROJECTILE_POWER_DELAY I
	FIELD field_31264 REGULAR_POWER_DELAY I
```

Comment (IotaBread) [reply mappings/net/minecraft/block/TargetBlock.mapping]: Maybe `REGULAR_ENTITY`?

Comment (liach) [reply mappings/net/minecraft/block/TargetBlock.mapping]: Just regular power delay, no need for `entity` as only entity can trigger targets

**Comment by liach** [comment]
wait a sec, no... persistent projectile is like arrow or trident that can be picked up after getting fired. the other delay is for other projectiles. So maybe we want to name it as `RECOVERABLE_DELAY` or similar names showing this is the distinguishing feature.

**Comment by IotaBread** [comment]
Oh yeah, you are right. I misread `PersistentProjectileEntity` as `ProjectileEntity` when checking the code

**Review by liach — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**